docs/troubleshoot.md

```diff
@@ -6,92 +6,93 @@ indent: true
-To get the current Crossplane logs, run the following:
+Most Crossplane resources have a `status` section that can represent the current
+state of that particular resource. Running `kubectl describe` against a
+Crossplane resource will frequently give insightful information about its
+condition.For example, to determine the status of a MySQLInstance resource
+claim, run:
 
-```console
-kubectl -n crossplane-system logs -lapp=crossplane
+```shell
+kubectl -n app-project1-dev describe mysqlinstance mysql-claim
 ```
 
-## Resource Status and Conditions
-
-All of the objects that represent managed resources such as databases, clusters, etc. have a `status` section that can give good insight into the current state of that particular object.
-In general, simply getting the `yaml` output of a Crossplane object will give insightful information about its condition:
+This should produce output that includes:
 
 ```console
-kubectl get <resource-type> -o yaml
+Status:
+  Conditions:
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Managed claim is waiting for managed resource to become bindable
+    Status:                False
+    Type:                  Ready
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Successfully reconciled managed resource
+    Status:                True
+    Type:                  Synced
 ```
 
-For example, to get complete information about an Azure AKS cluster object, the following command will generate the below sample (truncated) output:
+Most Crossplane resources set exactly two condition types; `Ready` and `Synced`.
+`Ready` represents the availability of the resource itself - whether it is
+creating, deleting, available, unavailable, binding, etc. `Synced` represents
+the success of the most recent attempt to 'reconcile' the _desired_ state of the
+resource with its _actual_ state.
 
-```console
-> kubectl -n crossplane-system get akscluster -o yaml
-...
-  status:
-    Conditions:
-    - LastTransitionTime: 2018-12-04T08:03:01Z
-      Message: 'failed to start create operation for AKS cluster aks-demo-cluster:
-        containerservice.ManagedClustersClient#CreateOrUpdate: Failure sending request:
-        StatusCode=400 -- Please see https://aka.ms/acs-sp-help for more details."'
-      Reason: failed to create cluster
-      Status: "False"
-      Type: Failed
-    - LastTransitionTime: 2018-12-04T08:03:14Z
-      Message: ""
-      Reason: ""
-      Status: "False"
-      Type: Creating
-    - LastTransitionTime: 2018-12-04T09:59:43Z
-      Message: ""
-      Reason: ""
-      Status: "True"
-      Type: Ready
-    bindingPhase: Bound
-    endpoint: crossplane-aks-14af6e93.hcp.centralus.azmk8s.io
-    state: Succeeded
-```
+## Crossplane Logs
+
+The next place to look to get more information or investigate a failure would
+be in the Crossplane pod logs, which should be running in the
+`crossplane-system` namespace. To get the current Crossplane logs, run the
```

Comment:
given that we have introduced an `infra` namespace in stack and services guide, we should also probably mention that here instead of `crossplane-system`

Reply:
This is discussing pod logs for the actual crossplane process, which I we still run in `crossplane-system`. We should probably expand this to mention that you might want to look at stack logs too though.